tools/setup/emoji/emoji_names.py

```diff
@@ -46,7 +46,7 @@
     '1f913': {'canonical_name': 'nerd', 'aliases': ['geek']},
     # several sites suggested this was used for "cool", but cool is taken by
     # Symbols/137
-    '1f60e': {'canonical_name': 'sunglasses', 'aliases': []},
+    '1f60e': {'canonical_name': 'sunglasses', 'aliases': ['awesome', 'cool']},
```

Comment:
as per the conversation on czo, le'ts keep 'cool' but remove 'awesome'

Reply:
```suggestion
    '1f60e': {'canonical_name': 'sunglasses', 'aliases': ['cool']},
```
done